tests/mock_crust/churn.rs

```diff
@@ -201,64 +317,126 @@ fn verify_section_list_signatures(nodes: &[TestNode]) {
-        expected_gets.send_and_expect(data_id, auth_n0, auth_n0, &mut nodes, min_section_size);
-        expected_gets.send_and_expect(data_id, auth_n0, auth_n1, &mut nodes, min_section_size);
-        expected_gets.send_and_expect(data_id, auth_n0, auth_g0, &mut nodes, min_section_size);
-        expected_gets.send_and_expect(data_id, auth_n0, auth_s0, &mut nodes, min_section_size);
-        // ... and from a group to itself, another group, a section and a node...
-        expected_gets.send_and_expect(data_id, auth_g0, auth_g0, &mut nodes, min_section_size);
-        expected_gets.send_and_expect(data_id, auth_g0, auth_g1, &mut nodes, min_section_size);
-        expected_gets.send_and_expect(data_id, auth_g0, auth_s0, &mut nodes, min_section_size);
-        expected_gets.send_and_expect(data_id, auth_g0, auth_n0, &mut nodes, min_section_size);
-        // ... and from a section to itself, another section, a group and a node...
-        // TODO: Enable these once MAID-1920 is fixed.
-        // expected_gets.send_and_expect(data_id, auth_s0, auth_s0, &nodes, min_section_size);
-        // expected_gets.send_and_expect(data_id, auth_s0, auth_s1, &nodes, min_section_size);
-        // expected_gets.send_and_expect(data_id, auth_s0, auth_g0, &nodes, min_section_size);
-        // expected_gets.send_and_expect(data_id, auth_s0, auth_n0, &nodes, min_section_size);
-
-        // Test messages from a client to a group and a section...
-        expected_gets.client_send_and_expect(data_id, cl_auth, auth_g0, &clients[0], &mut nodes);
-        expected_gets.client_send_and_expect(data_id, cl_auth, auth_s0, &clients[0], &mut nodes);
-        // ... and from group to the client
-        expected_gets.send_and_expect(data_id, auth_g1, cl_auth, &mut nodes, min_section_size);
-
-        poll_and_resend(&mut nodes, &mut clients);
-
-        expected_gets.verify(&mut nodes, &mut clients);
+    let mut nodes = if initial_nodes == 1 {
+        Nodes(vec![TestNode::builder(&network).first().endpoint(Endpoint(0)).create()])
+    } else {
+        create_connected_nodes(&network, initial_nodes)
+    };
 
+    while nodes.len() < min_section_size {
+        let (added_index, _) = add_random_node(&mut rng, &network, &mut nodes, min_section_size);
+        let _ = poll_all(&mut nodes, &mut []);
         verify_invariant_for_all_nodes(&nodes);
-        verify_section_list_signatures(&nodes);
+        let section_size = nodes.len();
+        send_and_receive(&mut rng, &mut nodes, section_size, Some(added_index));
     }
 }
+
+#[test]
+fn bootstrap_1() {
+    bootstrap_from(1);
+}
+
+#[test]
+fn bootstrap_2() {
+    bootstrap_from(2);
+}
+
+#[test]
+fn bootstrap_3() {
+    bootstrap_from(3);
+}
+
+#[test]
+fn bootstrap_4() {
+    bootstrap_from(4);
+}
+
+#[test]
+fn bootstrap_5() {
+    bootstrap_from(5);
+}
+
+#[test]
+fn bootstrap_6() {
+    bootstrap_from(6);
+}
+
+#[test]
+fn bootstrap_7() {
+    bootstrap_from(7);
+}
```

Comment:
Aren't those `bootstrap_` tests just duplicating the `test_nodes` tests in `tests/mock_crust/mod.rs`?